packages/perseus/src/types.ts

```diff
@@ -145,7 +146,7 @@ export type APIOptions = Readonly<{
         // These two parameters are only used in library-exercise.jsx and
         // practice-exercise.jsx and in the mobile app
         // (javascript/perseus-package/perseus-mobile-bridge.js).
```

Comment:
I kinda feel like we should just remove this comment. The first two components don't exist anymore, or are deprecated. Now that Perseus is back in its own repo, I think we can work under the assumption that any public API needs to be supported... if we drop some part of it, it's a major version change and consuming apps will need to be adjusted.